test/e2e/e2e_test.go

```diff
@@ -333,3 +333,27 @@ func verifyResources(t *testing.T, awaitility *wait.Awaitility, murName string,
 func toIdentityName(userID string) string {
 	return fmt.Sprintf("%s:%s", "rhd", userID)
 }
+
+// openShift3XConsoleURL checks if <apiEndpoint>/console URL is reachable.
+// This URL is used by web console in OpenShift 3.x
+func openShift3XConsoleURL(apiEndpoint string) string {
+	client := http.Client{
+		Timeout: time.Duration(1 * time.Second),
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		},
+	}
+	url := fmt.Sprintf("%s/console", apiEndpoint)
+	resp, err := client.Get(url)
+	if err != nil {
+		return ""
+	}
+	if resp.StatusCode != http.StatusOK {
+		return ""
+	}
+	defer func() {
+		_, _ = ioutil.ReadAll(resp.Body)
+		_ = resp.Body.Close()
+	}()
```

Comment:
shouldn't that be moved after the `Get` call?

Reply:
Yes it should :)

Reply:
Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/51/commits/fa602c224831eff6e45ae93f94aba7670a238af7